src/js/tracks/text-track-settings.js

```diff
@@ -327,7 +455,20 @@ class TextTrackSettings extends Component {
   }
 
   /**
-   * Restore text track settings
+   * Sets all <select> elements to their default values.
+   *
+   * @method setDefaults
+   */
+  setDefaults() {
+    Obj.each(selectConfigs, config => {
```

Comment:
We're iterating over `selectConfigs`, though. If anything, it'd be `Obj.each(selectConfigs, (config) =>`.